src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,192 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user = new User(RegistrationServiceTestConstants.OVER_MIN_LOGIN,
+                RegistrationServiceTestConstants.OVER_MIN_PASSWORD,
+                RegistrationServiceTestConstants.OVER_MIN_AGE);
+    }
+
+    @Test
+    void userIsAlreadyRegistered_NotOk() {
```

Comment:
please pay attention to your test methods namings. Let's follow the next naming convention : `<methodUnderTest>_<state>_<expectedBehavior>` Fix the problem in all places